src/test/java/seedu/address/logic/parser/FindByWorkExperienceCommandParserTest.java

```diff
@@ -32,4 +33,4 @@ public void parse_invalidArgs_throwsParseException() {
         assertParseFailure(parser, " ", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                 FindByWorkExperienceCommand.MESSAGE_USAGE));
     }
-}
\ No newline at end of file
+}
```

Comment:
Excellent work on keeping the method structure clean and readable. It’s easy to follow the logic.